src/app/factories/create-factory/template-tab/factory-from-template.controller.js

```diff
@@ -40,17 +40,26 @@ class FactoryFromTemplateCtrl {
     }
 
     this.isImporting = true;
-    // fetch it !
-    let promise = this.codenvyAPI.getFactoryTemplate().fetchFactoryTemplate(templateName);
 
-    promise.then((factoryContent) => {
-      this.isImporting = false;
-      this.factoryContent = this.$filter('json')(factoryContent, 2);
-    }, (error) => {
+    //TODO
+    // Templates are now stored in dashboard project
```

Comment:
why are they stored in dashboard ?
now we have a way to store them outside of dashboard so we can tweak them without rebuilding dashboard